command/agent/consul/client.go

```diff
@@ -1064,6 +1064,8 @@ func createCheckReg(serviceID, checkID string, check *structs.ServiceCheck, host
 		chkReg.TCP = net.JoinHostPort(host, strconv.Itoa(port))
 	case structs.ServiceCheckScript:
 		chkReg.TTL = (check.Interval + ttlCheckBuffer).String()
+		// As of Consul 1.0.0 setting TTL and Interval is a 400
```

Comment:
I am assuming this is true for TTL checks. It isn't immediately clear since the switch is for script check where interval and TTL seem totally valid. Can you add a comment that it is because we are using a TTL check underneath.